insights/config/specs.py

```diff
@@ -174,6 +174,8 @@
     "keystone.conf"             : SimpleFileSpec("etc/keystone/keystone.conf"),
     "keystone.log"              : SimpleFileSpec("var/log/keystone/keystone.log"),
     "keystone_crontab"          : CommandSpec("/usr/bin/crontab -l -u keystone"),
+    "krb5"                      : SimpleFileSpec("etc/krb5.conf"),
```

Comment:
It's also worth pointing out that once again we are faced with a program's configuration that is comprised of multiple files.  However, Krb5Configuration files will only ever see one file.  It will be up to the rule writer to make sure they parse all the files, and then they will need to resolve dependencies - e.g. default values from the [libdefaults] in /etc/krb5.conf affect a realm setting in /etc/krb5.conf.d/realm.conf.

The solution, at the moment, is to write a combiner that reads all the files and resolves them into one structure.  See the ``insights/combiners/limits_conf.py`` combiner for an example.

I proposed a MultiFileParser class that would take multiple files and provide a united interface to them but that was voted down.

I would really like to see this combiner provided as part of this commit, so that we do not end up with the situation where some rules are using the parser by itself and assuming that they don't have to deal with multiple files.

Reply:
@wushiqinlou - yes, Paul is right, we'd better declare them both "PatternSpec", Please refer to `limit.conf` and `limit.d`